File: 34-find-first-and-last-position-of-element-in-sorted-array/find-first-and-last-position-of-element-in-sorted-array.java
import java.util.ArrayList;

class Solution {
    public int[] searchRange(int[] nums, int target) {

        ArrayList<Integer> n = new ArrayList<>();

        for (int i = 0; i < nums.length; i++) {
            if (nums[i] == target) {
                n.add(i);
            }
        }

        if (n.size() > 0) {
            return new int[]{n.get(0), n.get(n.size() - 1)};
        }

        return new int[]{-1, -1};
    }
}

File: 34-find-first-and-last-position-of-element-in-sorted-array/find-first-and-last-position-of-element-in-sorted-array.py
class Solution:
    def searchRange(self, nums, target):

        n = []

        for i in range(len(nums)):
            if nums[i] == target:
                n.append(i)

        if len(n) > 0:
            return [n[0], n[-1]]

        return [-1, -1]